packages/@okta/vuepress-site/docs/concepts/auth-overview/index.md

```diff
@@ -286,26 +286,3 @@ app -> client: Response
 -->
 
 For information on how to set up your application to use this flow, see [Implement the Client Credentials Flow](/docs/guides/implement-client-creds/).
-
-## Authorization Servers
```

Comment:
We should keep this heading and include text with a link saying we now have a whole Concepts page dedicated to this topic.

Related: there is at least one link in our docs to this section, which will have to be updated to go to the new Concept page instead.